spec/pom.xml

```diff
@@ -14,8 +14,8 @@
 	<description>CDI Specification documentation</description>
 
 	<properties>
-		<asciidoctor-maven.version>1.5.6</asciidoctor-maven.version>
-		<asciidoctorj-pdf.version>1.5.0-alpha.14</asciidoctorj-pdf.version>
+		<asciidoctor-maven.version>2.2.1</asciidoctor-maven.version>
+		<asciidoctorj-pdf.version>1.6.0</asciidoctorj-pdf.version>
```

Comment:
This will break, other Asciidoctor configuration changes are required. At least based on reading the documentation (which I did for the SpecRef prototype). I specifically mean these changes: https://github.com/Ladicek/cdi-spec/blob/bb7cafddb2364dd78151a25c6a8d5a576baca0d0/spec/pom.xml#L212-L219 (without the `sourcemap` attribute, that was only for my prototype).

Reply:
Curious, I was having problems with this even building the pdf under SE 11 with the current version. I'll try just updating the asciidoctorj-pdf.

Reply:
No, I'm saying the `sourcemap` attribute is not necessary to migrate to new Asciidoc. The `imagesdir` and `source-highlighter` attributes have to be changed. I also added `sourceDirectory`, which isn't strictly necessary, but without it, you get semi-warnings about non-existing directories.